EnglishFilter/sections/css_extended.txt

```diff
@@ -972,6 +972,8 @@ tube-bunny.com#?#span:contains(Advertisement)
 !
 vrsumo.com#?#aside div.box-container:has(> div.inner-box-container a[target="_blank"]:contains(Ads by))
 flyflv.com#?#.box > div.content:has(> div.line > a#joinLink)
+speed-down.org#?#center > div.download-option-btn:has(> button.btn > a[href^="https://speed-down.org/download"])
```

Comment:
@BlazDT 
This element is not fully blocked by the rule `speed-down.org##a[href^="https://speed-down.org/download"]`, I think that it is better to change it to rule above

<details>
<summary>Element</summary>
<img src="https://user-images.githubusercontent.com/18536328/88315677-45fb5180-cd1f-11ea-98ad-ecafd68b204e.png">
</details>

Reply:
I added that simple rule for products which do not support ext-CSS rules.
With the ext-CSS rule I added, I do not see left-over.
`.download-option-btn > .btn-success:not(.downloadbtn):has(> button > a[href^="https://speed-down.org/download"])`

Reply:
Ok, now I see that ext-css rule works